tests/test_fluent_session.py

```diff
@@ -220,3 +220,22 @@ def test_interrupt(load_static_mixer_case):
     time.sleep(5)
     solver.solution.run_calculation.interrupt()
     assert solver.scheme_eval.scheme_eval("(rpgetvar 'time-step)") < 100
+
+
+def test_fluent_exit():
+    inside_container = os.getenv("PYFLUENT_LAUNCH_CONTAINER")
+    script = (
+        "import ansys.fluent.core as pyfluent;"
+        "solver = pyfluent.launch_fluent(start_watchdog=False);"
+        f'{"print(solver.connection_properties.cortex_host);" if inside_container else "print(solver.connection_properties.cortex_pid);"}'
+        "exit()"
+    )
```

Comment:
And I think will need to disable logging, in case it is active:
```suggestion
    script = (
        "import ansys.fluent.core as pyfluent;"
        "import os;"
        "os.environ.pop('PYFLUENT_LOGGING', None);"
        "os.environ.pop('PYFLUENT_WATCHDOG_DEBUG', None);"
        "solver = pyfluent.launch_fluent(start_watchdog=False);"
        f'{"print(solver.connection_properties.cortex_host);" if inside_container else "print(solver.connection_properties.cortex_pid);"}'
        "exit()"
    )
```

Reply:
Note I edited my suggestion above, `import ansys.fluent.core as pyfluent` needs to come after disabling logging for this command